src/app/components/footer/footer.component.ts

```diff
@@ -0,0 +1,44 @@
+import {Component, ElementRef, OnInit, ViewChild} from '@angular/core';
+import {faCheck} from '@fortawesome/free-solid-svg-icons';
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.css']
+})
+export class FooterComponent implements OnInit {
+
+  faCheck = faCheck;
+  @ViewChild('subscribeButton') subscribeButton: ElementRef;
+  @ViewChild('spinner') spinner: ElementRef;
+  @ViewChild('buttonText') buttonText: ElementRef;
+  @ViewChild('emailInput') emailInput: ElementRef;
+  @ViewChild('checkMark') checkMark: ElementRef;
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+  subscribeToNewsletter() {
+
+    if (/^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/.test(this.emailInput.nativeElement.value)) {
+      this.spinner.nativeElement.removeAttribute('hidden');
+      this.buttonText.nativeElement.setAttribute('hidden', 'true');
+      this.subscribeButton.nativeElement.setAttribute('disabled', 'true');
+
+      console.dir(this.emailInput.nativeElement.value);
```

Comment:
You shold get the content of the element with the [NgModel Directive](https://angular.io/api/forms/NgModel).